tests/acl/test_stress_acl.py

```diff
@@ -13,115 +16,162 @@
+                            prepare_test_port, get_function_conpleteness_level,
+                            toggle_all_simulator_ports_to_rand_selected_tor):   # noqa F811
+
+    ptf_src_port, ptf_dst_ports, dut_port = prepare_test_port
+
+    cmd_create_table = "config acl add table STRESS_ACL L3 -s ingress -p {}".format(dut_port)
+    cmd_remove_table = "config acl remove table STRESS_ACL"
     cmd_add_rules = "sonic-cfggen -j {} -w".format(STRESS_ACL_RULE_JSON_FILE)
-    cmd_rm_rules = "acl-loader delete STRESS_ACL"
+    cmd_readd_rules = "sonic-cfggen -j {} -w".format(STRESS_ACL_READD_RULE_JSON_FILE)
+    cmd_rm_all_rules = "acl-loader delete STRESS_ACL"
 
-    loganalyzer = LogAnalyzer(ansible_host=rand_selected_dut, marker_prefix="stress_acl")
-    loganalyzer.match_regex = [LOG_EXPECT_ACL_RULE_FAILED_RE]
-    try:
-        logger.info("Creating ACL rules in STRESS_ACL")
-        with loganalyzer:
-            rand_selected_dut.shell(cmd_add_rules)
-    except LogAnalyzerError as err:
-        # Cleanup Config DB if failed
-        logger.error("ACL rule creation failed, attempting to clean-up...")
-        rand_selected_dut.shell(cmd_rm_rules)
-        raise err
-    yield
-    # Remove testing rules
-    logger.info("Removing testing ACL rules")
-    rand_selected_dut.shell(cmd_rm_rules)
-
-
-def test_acl_add_del_stress(rand_selected_dut, setup_stress_acl_rules, get_function_conpleteness_level,
-                            toggle_all_simulator_ports_to_rand_selected_tor):   # noqa F811
     normalized_level = get_function_conpleteness_level
     if normalized_level is None:
         normalized_level = 'basic'
     loop_times = LOOP_TIMES_LEVEL_MAP[normalized_level]
+    wait_time = 2
+
+    rand_selected_dut.shell(cmd_create_table)
+    acl_rule_list = list(range(1, ACL_RULE_NUMS + 1))
+    verify_acl_rules(rand_selected_dut, ptfadapter, ptf_src_port, ptf_dst_ports, acl_rule_list, 0, "forward")
 
-    with open(STRESS_ACL_BASH_TEMPLATE, 'r') as f:
-        file_data = ""
-        for line in f:
-            if "loop_times=" in line:
-                line = "loop_times={}\n".format(loop_times)
-            file_data += line
-    with open(STRESS_ACL_BASH_TEMPLATE, 'w') as f:
-        f.write(file_data)
-
-    rand_selected_dut.template(src=STRESS_ACL_BASH_TEMPLATE, dest=STRESS_ACL_BASH_FILE)
-    rand_selected_dut.shell("nohup bash {} > {} 2>&1 &".format(STRESS_ACL_BASH_FILE, STRESS_ACL_BASH_LOG_FILE))
-    ask_times = loop_times / 10
-    while ask_times >= 0:
-        output = rand_selected_dut.shell('ps -aux|grep bash')['stdout']
-        wait(20, 'Wait some time to check if bash script finished.')
-        ask_times -= 1
-        if STRESS_ACL_BASH_FILE in output:
-            continue
+    loops = 0
+    while loops < loop_times:
+        logger.info("loops: {}".format(loops))
+        if loops == 0:
+            rand_selected_dut.shell(cmd_add_rules)
```

Comment:
Rewriting the same json file is actually updating the existing rules. I suggest appending new rules instead of updating existing rules.